### ISSUE TITLE
Documentation Component in last line does not get rendered

### ISSUE THREAD

**Issue by ChristophLabacher** [atelierdisko/dsk#100]
When a documentation component is in the last line of a document it does not get displayed by the frontend. 

For an example see `playground.md` of Banner:

## Not on last line:

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/7572627/63160995-5d19c580-c01f-11e9-86bc-8f0e7fa0b1f5.png">

<img width="1775" alt="image" src="https://user-images.githubusercontent.com/7572627/63161015-66a32d80-c01f-11e9-92ff-25a30e586e83.png">

## On last line:

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/7572627/63161033-6f93ff00-c01f-11e9-9a53-ad2f99f3035f.png">

<img width="1773" alt="image" src="https://user-images.githubusercontent.com/7572627/63161053-7b7fc100-c01f-11e9-96ff-c78b04425311.png">

## On last line with other content before:
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/7572627/63161078-89cddd00-c01f-11e9-8d1b-4f6bf16338dc.png">

<img width="1763" alt="image" src="https://user-images.githubusercontent.com/7572627/63161090-8df9fa80-c01f-11e9-9f7d-47b6ffc4dc92.png">

**Comment by ChristophLabacher** [comment]
This appears to be a backend problem: For some reason the component in the last line is still preceded by an opening `<script>`-tag:

Document: 

```md
a

<Banner title="Banner" type="warning">Use banners to highlight things people shouldn’t miss.</Banner>

<Banner title="Banner" type="warning">Use banners to highlight things people shouldn’t miss.</Banner>
```

API Response for document (html)
```html
<p>a</p>

<Banner title="Banner" type="warning">Use banners to highlight things people shouldn’t miss.</Banner>

<p><script>&rsquo;|dsk|&rsquo;<Banner title="Banner" type="warning">Use banners to highlight things people shouldn’t miss.</Banner></p>
```

**Comment by mariuswilms** [comment]
Fixed in b7abed2, will ship with next 1.2 release.
